system_tests/bigtable.py

```diff
@@ -227,3 +230,65 @@ def test_rename_table(self):
         self.assertEqual(
             exc_caught.details,
             'BigtableTableService.RenameTable is not yet implemented')
+
+    def test_create_column_family(self):
+        temp_table_id = 'foo-bar-baz-table'
```

Comment:
I like to give resources in my tests somewhat meaningful names so in the event of a bad state of the world I can more easily track down the test.

No action required here, just a note.